tests/page/page-history.spec.ts

```diff
@@ -53,28 +53,40 @@ it('page.goBack should work with HistoryAPI', async ({ page, server }) => {
 });
 
 it('page.goBack should work for file urls', async ({ page, server, asset, browserName, platform, isAndroid }) => {
-  it.fail(browserName === 'webkit' && platform === 'darwin');
+  it.fixme(browserName === 'firefox', 'Firefox pretends, but does not complete goBack to the file url');
+  it.fail(browserName === 'webkit' && platform === 'darwin', 'WebKit embedder fails to go back/forward to the file url.');
```

Comment:
make it fixme, otherwise it always times out?

Reply:
WebKit is failing with an exception, so we are good. Leaving as it was.